src/cdi_ndi/config.h.in

```diff
@@ -22,9 +22,9 @@
 #endif
 
 /* NDI version (if found) */
-#cmakedefine NDI_VERSION_STRING "@NDI_VERSION@"
```

Comment:
This was evaluating to `/* #undef NDI_VERSION_STRING */` because `NDI_VERSION_STRING` was not previously declared by cmake

Reply:
Your solution looks correct TO ME.